lldb/include/lldb/Utility/ConstString.h

```diff
@@ -192,6 +198,9 @@ class ConstString {
     return llvm::StringRef(m_string, GetLength());
   }
 
+  /// Get the string value as a std::string
+  std::string GetString() const { return std::string(m_string, GetLength()); }
```

Comment:
why is this necessary, if we have the explicit operator?

Reply:
It's not necessary, it's more for consistency with how we already do things (similar to how `StringRef` has the explicit operator and the `.str()` method on the llvm side).